packages/connect/src/device/Device.ts

```diff
@@ -656,6 +662,7 @@ export class Device extends TypedEmitter<DeviceEvents> {
 
     async getFeatures() {
         const { message } = await this.getCurrentSession().typedCall('GetFeatures', 'Features', {});
+        await this._updateCurrentRelease(message);
```

Comment:
Basically main reason we need to have it here is because in `_updateFeatures` we do:

```
            this.availableTranslations = this._currentRelease?.translations ?? {};

```

And as it was done before it was geting only releases of bundled ones, but if we use it for a FW which is only provided in a remote way we need to fetch it. 

So `getReleaseByVersion` that is used inside `_updateCurrentRelease` gets the bundled one if present so it does not have to fetch anything in case of desktop but if it is not present in the bundled then it fetches it.